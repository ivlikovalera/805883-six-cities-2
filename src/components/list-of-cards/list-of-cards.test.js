import React from 'react';
import renderer from 'react-test-renderer';
import ListOfCards from './list-of-cards.jsx';

it(`List correctly renders after relaunch`, () => {

  const listComponent = renderer
  .create(<ListOfCards
    places={[]}
  />)
  .toJSON();

  expect(listComponent).toMatchSnapshot();
});
