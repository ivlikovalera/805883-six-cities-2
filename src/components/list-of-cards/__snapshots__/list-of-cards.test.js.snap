// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List correctly renders after relaunch 1`] = `
<div
  className="near-places__list places__list"
/>
`;
